src/pages/our-work.js

```diff
@@ -92,7 +101,7 @@ const OurWork = ({ data }) => {
       </Grid>
       <GreyBackground>
         <Grid>
-          {allCaseStudies.map((caseStudy, index, arr) => {
+          {orderedCaseStudies.map((caseStudy, index, arr) => {
```

Comment:
I would `orderedCaseStudies && orderedCaseStudies > 0 && ....` here just for peace of mind